app/assets/stylesheets/roster.scss

```diff
@@ -66,7 +66,7 @@
 
 .highcharts-container {
  overflow: visible !important;
- margin-top: 60px !important;
+ margin-top: 10px !important;
```

Comment:
Changing this will have effects on the classroom roster page as well.  One approach could be adding another SCSS file for the profile page, and scoping this rule to only the profile page.  Or you could verify that this change looks fine on the classroom roster page and drop a screenshot of that too.